Input/InputFieldSpotlightDecorator.js

```diff
@@ -201,8 +206,14 @@ const InputSpotlightDecorator = hoc(defaultConfig, (config, Wrapped) => {
 			}
 		};
 
-		focus = (focused, node) => {
-			this.setState({focused, node});
+		focus = (focused, node, fromMouse) => {
+			this.setState(() => (
+				{
+					focused,
+					node,
+					fromMouse: fromMouse
```

Comment:
```suggestion
					fromMouse
```

Reply:
I think we can just add `fromMouse` next to the `node`.

Reply:
Thank you. And I changed it to not use the function like the previous one.